.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**What distribution you're using and the version.**
+
+
+**What spl/zfs packages you're using and the version.**
```

Comment:
I'd reword this and previous to be `you're` -> `are you`. You are reads like a directive, but you are asking a question.